deepchem/models/torch_models/infograph.py

```diff
@@ -223,7 +231,8 @@ def __init__(self, encoder, unsup_encoder, ff1, ff2, fc1, fc2, local_d,
         self.fc2 = fc2
         self.local_d = local_d
         self.global_d = global_d
-        self.init_emb()
+        if init_emb:
+            self.init_emb()
```

Comment:
This change is made so that .restore() functions as expected. Otherwise the bias weights will be filled with 0.